cirq/google/engine/engine.py

```diff
@@ -737,15 +737,16 @@ def get_calibration(self, calibration_name: str) -> calibration.Calibration:
 
         Params:
             calibration_name: A string of the form
-                `<processor name>/calibrations/<ms since epoch>`
+                `projects/<project_id>/processors/<processor name>`
+                `/calibrations/<ms since epoch>`
```

Comment:
The original doc using <processor name> is correct (except for the timestamp units) but I'm not certain what convention is being used here.
The full resource name for a calibration object is:
`projects/<project id>/processors/<processor id>/calibrations/<timestamp in seconds since epoch>`

Reply:
Yeah, it looks like you need the whole resource name or else it fails.  Good catch on the units.